nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<blockquote>that have are preceeded</blockquote><p> </p><p>I think should be "that are preceeded"</p><p> </p><blockquote>that do some kind of</blockquote><p> </p><p>Could be simplified to "and do"</p><p> </p><p>I would link to some python docs about the dunder all export. Most people I've talked to actually don't know about this, and this is a great opportunity to educate people by placing a link on that bullet point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>